cmd/edv-rest/startcmd/start.go

```diff
@@ -132,6 +132,18 @@ const (
 		"Defaults to false if not set. " + commonEnvVarUsageText + authEnableEnvKey
 	authEnableEnvKey = "EDV_AUTH_ENABLE"
 
+	returnFullDocumentOnQueryExtensionName = "ReturnFullDocumentsOnQuery"
+	readAllDocumentsExtensionName          = "ReadAllDocuments"
```

Comment:
We don't actually use the read all documents endpoint, but it's something I implemented awhile back and it might make its way into the spec eventually: https://github.com/decentralized-identity/confidential-storage/issues/111